packages/terra-date-time-picker/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Updated wdio screenshots due to changes in react-datepicker colors.
+
+4.27.0 - (February 4, 2020)
```

Comment:
Looks like this line should not be here.

Reply:
Oops, removed e4930e6